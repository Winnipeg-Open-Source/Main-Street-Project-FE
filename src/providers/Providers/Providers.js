import React from 'react';
import PropTypes from 'prop-types';
import ThemeProvider from 'providers/ThemeProvider';
import ResponsiveProvider from 'providers/Responsive';
import SavingProvider from 'providers/Saving';
import SidebarProvider from 'providers/Sidebar';
import DonorsProvider from 'providers/Donors';

function Providers ({ children }) {
    return (
        <ThemeProvider>
            <ResponsiveProvider>
                <SavingProvider>
                    <SidebarProvider>
                        <DonorsProvider>
                            {children}
                        </DonorsProvider>
                    </SidebarProvider>
                </SavingProvider>
            </ResponsiveProvider>
        </ThemeProvider>
    );
}

Providers.displayName = 'Providers';

Providers.propTypes = {
    children: PropTypes.node,
};

export default Providers;
