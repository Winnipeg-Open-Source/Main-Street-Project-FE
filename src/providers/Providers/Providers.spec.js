import React from 'react';
import { render } from '@testing-library/react';
import Providers from './Providers';

describe ('Providers', () => {
    it ('renders without crashing', () => {
        const { asFragment } = render(<Providers />);

        expect(asFragment()).toMatchSnapshot();
    });
});
