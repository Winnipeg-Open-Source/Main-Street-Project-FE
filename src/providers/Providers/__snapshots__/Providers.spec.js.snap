// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Providers renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="sc-pbJYR hNzbLI sc-pkfPC fCxHK"
  />
</DocumentFragment>
`;
